qemu/tests/boot_with_different_vectors.py

```diff
@@ -46,42 +45,29 @@ def boot_guest_with_vectors(vectors):
         vm.verify_alive()
         return vm
 
-    def enable_multi_queues(vm):
-        session = vm.wait_for_login(timeout=login_timeout)
-        error_context.context("Enable multi queues in guest.", logging.info)
-        for nic_index, nic in enumerate(vm.virtnet):
-            ifname = utils_net.get_linux_ifname(session, nic.mac)
-            queues = int(nic.queues)
-            mq_set_cmd = "ethtool -L %s combined %d" % (ifname, queues)
-            status, output = session.cmd_status_output(mq_set_cmd)
-            if status:
-                msg = "Fail to enable multi queue in guest."
-                msg += "Command %s, fail with output %s" % (mq_set_cmd, output)
-                test.error(msg)
-
     def check_msi_support(session):
-        devices = session.cmd_output("lspci |grep Eth").strip()
+        devices = session.cmd_output("lspci | grep Eth").strip()
         vectors = params["vectors"]
         error_context.context("Check if vnic inside guest support msi.",
                               logging.info)
         for device in devices.split("\n"):
             if not device:
                 continue
             d_id = device.split()[0]
-            msi_check_cmd = "lspci -vvv -s %s |grep MSI" % d_id
+            msi_check_cmd = "lspci -vvv -s %s | grep MSI" % d_id
             output = session.cmd_output(msi_check_cmd)
             if vectors == 0 and output:
                 test.fail("Guest do not support msi when vectors = 0.")
             if output:
                 if vectors == 1:
-                    if "MSI-X: Enable-" in output:
+                    if output.find("MSI-X: Enable-"):
```

Comment:
This is not correct.

string.find()
> Return the lowest index in s where the substring sub is found such that sub is wholly contained in s[start:end]. Return -1 on failure.

So when `"MSI-X: Enable-"` not in the `output` it returns `-1`. 
```
>>> if -1:
...    print(-1)
... 
-1
```